sympy/integrals/meijerint.py

```diff
@@ -604,6 +604,10 @@ def _condsimp(cond):
         (Or(p < q, Eq(p, q)), p <= q),
         # The next two obviously are instances of a general pattern, but it is
```

Comment:
It's not "the next two" anymore, I count four

Reply:
Well, I think it as two for `arg`, two for `unbranched_argument`.

Reply:
It may turn out that the latter pair can be removed.